package.json

```diff
@@ -26,11 +26,17 @@
   "devDependencies": {
     "@automattic/custom-colors-loader": "automattic/custom-colors-loader",
     "@automattic/dops-components": "automattic/dops-components",
-    "babel-core": "5.8.25",
+    "babel-core": "^6.8.0",
```

Comment:
Did you intend to change from explicit versions to version ranges with the caret prefix?

If you didn't and want to avoid this in the future, you can add a `.npmrc` file as we've done in Calypso with `save-exact`: https://github.com/Automattic/wp-calypso/blob/master/.npmrc